modules/claims_api/app/sidekiq/claims_api/report_monthly_submissions.rb

```diff
@@ -3,24 +3,51 @@
 module ClaimsApi
   class ReportMonthlySubmissions < ClaimsApi::ReportingBase
     def perform
-      if Settings.claims_api.report_enabled
-        @to = Time.zone.now
-        @from = 1.month.ago
-        pact_act_data = ClaimsApi::ClaimSubmission.where(created_at: @from..@to)
-        @claims_consumers = ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to).pluck(:cid).uniq
-        @poa_consumers = ClaimsApi::PowerOfAttorney.where(created_at: @from..@to).pluck(:cid).uniq
-        @itf_consumers = ClaimsApi::IntentToFile.where(created_at: @from..@to).pluck(:cid).uniq
-        @ews_consumers = ClaimsApi::EvidenceWaiverSubmission.where(created_at: @from..@to).pluck(:cid).uniq
-
-        ClaimsApi::SubmissionReportMailer.build(
-          @from,
-          @to,
-          pact_act_data,
-          consumer_claims_totals: monthly_claims_totals,
-          poa_totals:,
-          itf_totals:,
-          ews_totals:
-        ).deliver_now
+      return unless Settings.claims_api.report_enabled
+
+      @from = 1.month.ago
+      @to = Time.zone.now
+      @claims_consumers = ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to).pluck(:cid).uniq
+      @poa_consumers = ClaimsApi::PowerOfAttorney.where(created_at: @from..@to).pluck(:cid).uniq
+      @itf_consumers = ClaimsApi::IntentToFile.where(created_at: @from..@to).pluck(:cid).uniq
+      @ews_consumers = ClaimsApi::EvidenceWaiverSubmission.where(created_at: @from..@to).pluck(:cid).uniq
+
+      ClaimsApi::SubmissionReportMailer.build(
+        @from,
+        @to,
+        consumer_claims_totals: monthly_claims_totals,
+        poa_totals:,
+        itf_totals:,
+        ews_totals:
+      ).deliver_now
+    end
+
+    private
+
+    def monthly_claims_totals
+      monthly_claims_consumers = ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to)
+      monthly_pact_claims = ClaimsApi::ClaimSubmission.where(created_at: @from..@to, claim_type: 'PACT',
+                                                             claim_id: monthly_claims_consumers.pluck(:id))
```

Comment:
Anyone with access to mock data: 

Is `claim_type: 'PACT'` a necessary condition, or can we assume that any `ClaimSubmission` is a PACT claim?

Reply:
On second thought, I am removing the `claim_type: 'PACT'` constraint since we did not have it before.